clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/vmimage/view/AzureVMImageLookupController.groovy

```diff
@@ -61,15 +62,31 @@ class AzureVMImageLookupController {
   List<AzureNamedImage> getVMImage(@PathVariable String account, @PathVariable String region, @PathVariable String imageId) {
```

Comment:
Triggering the pipeline to create a new servergroup (Azure VMSS) we will use the name of the image passed in in order to find its other details (URI, OS type). The search is for the exact name while the other "find" entry is for all the images containing the passed name; we could have the deck sort through the returned list though. 

BTW there are three categories of images we expect the user to select from in the deck:
- custom images (VHDs) from Azure storage blobs that the user/app administrator specified in the config.yml file
- select Azure Market Store VM images that the  user/app administrator specified in the config.yml file
- any available Azure Market Store VM images only if find also passes "configOnly=0" and "q.length() > 3" and image name contains "q"